Update application-gateway-end-to-end-ssl-powershell.md

mistermik opened the pull request:
Added n.2 not described scenarios:
1- How to renew an expired Backend HTTPS certificate with Powershell (AZ module)
2-How to remove expired Backend certificate with Powershell (AZ module)

Commit:
Update application-gateway-end-to-end-ssl-powershell.md

Added n.2 not described scenarios:
1- How to renew an expired Backend HTTPS certificate with Powershell (AZ module)
2-How to remove expired Backend certificate with Powershell (AZ module)

vhorne reviewed (changes requested):
Please change the introductory sentence to the following:
"Use this procedure to apply a new certificate if the backend certificate it is expired."

vhorne reviewed (changes requested):
Please change the second introductory sentence to the following:

"Use this procedure to remove an unused expired certificate from HTTP Settings."

Commit:
Update application-gateway-end-to-end-ssl-powershell.md

Committed suggestion by vhore, changed introductory sentence to the following: "Use this procedure to apply a new certificate if the backend certificate it is expired." and changed the second introductory sentence to the following:
"Use this procedure to remove an unused expired certificate from HTTP Settings."

mistermik reviewed (commented):
Committed suggestion by vhorne, changed introductory sentence to the following: "Use this procedure to apply a new certificate if the backend certificate it is expired." and changed the second introductory sentence to the following:
"Use this procedure to remove an unused expired certificate from HTTP Settings."

GitHubber17 committed:
adjust text and list format
